### PR TITLE
feat(footer): added footer

### LINKED ISSUE #22: [Feat]: Add a Responsive Footer
## Description
[Initial Designs](https://www.figma.com/design/1nsYgWIN5bC7NnnVApTcK4/Salinas---bizzNest-Scheduler?node-id=0-1&node-type=canvas&t=TLQ1O5AYgPPOLJtm-0) design

### Web
image

### Mobile
image

> \[!NOTE]\
>  Every single UI PR should consist of a Screenshot and/or video (You can use cmd+shift+4 for screenshots and cmd+shift+5 for video recordings) and should be mobile responsive (show the mobile view)

### Tasks

- [x] Create a footer with proper semantics tags
- [x] Have the correct background color
- [x] Have the correct font
- [ ] Have the Digital Nest Logo 
- [x] Have the content properly aligned (keep title center)

### Additional Resources + Notes
Good to look into CSS + Semantic Tags + HTML

### PR DESCRIPTION
**Pull request by lilianaconstantino**
**When you open your pull requests, remove the lines that start and end with underscores (\_)**

## Changes
Added logo to footer 
Screen Shot 2024-09-25 at 12 53 26 PM

Fixed mobile styles 
Screen Shot 2024-09-25 at 12 51 26 PM

Closes #22

### CODE REVIEW

**Review by jacky-p — APPROVED** [review]
Looks good!

File: index.html [hunk 1]
```diff
@@ -25,6 +25,11 @@ <h1 class="name">Nest Meet</h1>
 
       <div class="header-spacer"></div>
     </div>
-    
+   
+
+    <footer class = "Digital-Nest-Footer">
```
Comment (elimanzo) [inline index.html:30]: lets keep class names kebab-casing and leave it lower case
```suggestion
    <footer class="digital-nest-footer">
```

File: src/style.css [hunk 2]
```diff
@@ -1,8 +1,29 @@
...
+  align-items: center; 
+  justify-content: center;
+  background-color: #BDE3D0;
+  gap: 20px;
+}
+
+  footer img {
+    width: 30px;
+    height: 30px;
+    }
```
Comment (elimanzo) [inline src/style.css:23]: There seems to be unnecessary spacing for this. Remind me to see if prettier is working for everyone :)

```suggestion
footer img {
  width: 30px;
  height: 30px;
}
```

**Review by elimanzo — CHANGES_REQUESTED** [review]
Hey @lilianaconstantino good job for the Footer Component. Just small nits, and this is ready to go!

**Review by elimanzo — APPROVED** [review]
LGTM!

**Pull request merged**